createdisk.sh

```diff
@@ -51,7 +51,7 @@ function create_qemu_image {
     ${QEMU_IMG} commit $destDir/${VM_PREFIX}-master-0
 
     # Check which partition is labeled as `root`
-    partition=$(${VIRT_FILESYSTEMS} -a $destDir/${VM_PREFIX}-base  -l | grep root | cut -f1 -d' ')
+    partition=$(${VIRT_FILESYSTEMS} -a $destDir/${VM_PREFIX}-base -l --partitions | sort -rk4 -n | sed -n 1p | cut -f1 -d' ')
```

Comment:
the ` sed -n 1p | cut -f1 -d` can alternatively be changed to `awk '{ if (NR == 1)  {print $1}  }'`. I'd at least use `head -1` instead of `sed -n 1p`

Reply:
>  I'd at least use head -1 instead of sed -n 1p

@cfergeau Is there any reason `head -1` better than `sed -n 1p` ?

Reply:
only (bad) reason is that I know what `head -1` is doing without looking at the man page, but I'm not familiar at all with `sed -n 1p`. No idea if it's just me, or if this is going to be typical :)

Reply:
@cfergeau I will not take as a reason then, you now know a new way to perform same thing 😉